main/ipc/AnalyzerChannel.ts

```diff
@@ -1,7 +1,7 @@
 import { IpcMainEvent } from 'electron';
 
 import { IpcRequest, IpcResponse } from '../../@types';
-import Analyzer from '../Analyzer';
+import Analyzer from '../analyzer';
```

Comment:
oops, my bad

Reply:
@julesyoungberg no worries, just Windows being stupid with path in-sensitivity.